packages/trader/src/Assets/Trading/Categories/__tests__/accumulator-trade-description.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { AccumulatorTradeDescription } from '../accumulator-trade-description';
+
+describe('<AccumulatorTradeDescription />', () => {
+    const trade_descriptions_paragraphs: string[] = [
```

Comment:
@henry-deriv let's not add the big blocks of text to the test.
Please check an example of adding types to this kind of component here: https://github.com/binary-com/deriv-app/blob/5d3a87af5c14a9c14a04714b8c1606b47edc89dd/packages/trader/src/Assets/Trading/Categories/__tests__/turbos-trade-description.spec.tsx

Reply:
mmm? This only has one line of text.